src/index.js

```diff
@@ -72,7 +79,8 @@ export default function remember(source) {
 
     sink(0, talkback)
 
-    if (inited && endValue === UNIQUE) {
+    if (inited && !greeted && endValue === UNIQUE) {
```

Comment:
TODO: quite not sure why `&& endValue === UNIQUE` is here, need to check this logic - it seems to me now that it shouldnt be here because sink wont be notified about remembered value, but maybe it was for some scenario around sync termination and it shouldnt be notified in such case? not sure right now, but also have to consider if if statement added above shouldnt have also similar condition